tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -317,11 +317,14 @@ def _setup_node(self) -> None:
         tarball_path: Path = Path("/tmp/waagent.tar")
         log.info("Creating %s with the files need on the test node", tarball_path)
         log.info("Adding orchestrator/scripts")
-        run_command(['tar', 'cvf', str(tarball_path), '--transform=s,.*/,bin/,', '-C', str(self.context.test_source_directory/"orchestrator"/"scripts"), '.'])
```

Comment:
Adding '.' to the tarball was overwriting the attributes of the home directory on the test machine (we extract the tarball with cwd set to $HOME). This was causing SSH issues on some distros, on which the SSH server checks the mode of $HOME and refuses the connection if it is too open.

Instead of adding '.', now we cd to the source directory and add '*'. The other 2 invocations of tar were OK, but I also changed them for consistency.